session/storage_memory.go

```diff
@@ -24,14 +24,16 @@ import (
 // NewStorageMemory initiates new session storage
 func NewStorageMemory() *StorageMemory {
 	return &StorageMemory{
-		sessionMap: make(map[ID]Session),
+		sessions:   make([]Session, 0),
+		sessionMap: make(map[ID]int),
 		lock:       sync.Mutex{},
 	}
 }
 
 // StorageMemory maintains a map of session id -> session
 type StorageMemory struct {
-	sessionMap map[ID]Session
```

Comment:
Was is worth it to remake it into array? Whats motivation?

Reply:
Because in `List()` I need list of items in as they were added with `Add()`